Index: LOGIDUTRA-master/src/main/java/org/example/Repository/CustomizerFactory.java
===================================================================
diff --git a/LOGIDUTRA-master/src/main/java/org/example/Repository/CustomizerFactory.java b/LOGIDUTRA-master/src/main/java/org/example/Repository/CustomizerFactory.java
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/java/org/example/Repository/CustomizerFactory.java	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,25 +0,0 @@
-
-package org.example.repositories;
-
-import jakarta.persistence.EntityManager;
-import jakarta.persistence.EntityManagerFactory;
-import org.hibernate.SessionFactory;
-import org.hibernate.cfg.Configuration;
-
-public class CustomizerFactory {
-    private static final EntityManagerFactory emf;/*emf é a variavel de EntityManagerFactory, o static é por que vai ser uma so pra toda a aplicação
-                                                  o final será inicializadao uma única vez e não pode ser alterada depois*/
-    public static EntityManager getEntityManager() {
-        return emf.createEntityManager();// aqui ele vai cria um entitymanager novo toda vez que for chamado
-    }
-
-    public static void fechar() {
-        emf.close();// aqui ele vai fecha o entitymanager, sempre deve fechar pra evitar vazamento de dados etc
-    }
-
-    static {
-        SessionFactory sessionFactory = (new Configuration()).configure("hibernate.cfg.xml").buildSessionFactory();//crias as conexoe com o hibernate
-        emf = (EntityManagerFactory)sessionFactory.unwrap(EntityManagerFactory.class);/*Converte o SessionFactory do Hibernate em um EntityManagerFactory (padrão JPA).
-                                                                                      pega do SessionFactory a parte que funciona como EntityManagerFactory*/
-    }
-}
Index: LOGIDUTRA-master/src/main/resources/db/migration/V3__create_motoristas.sql
===================================================================
diff --git a/LOGIDUTRA-master/src/main/resources/db/migration/V3__create_motoristas.sql b/LOGIDUTRA-master/src/main/resources/db/migration/V3__create_motoristas.sql
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/resources/db/migration/V3__create_motoristas.sql	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,7 +0,0 @@
-create table Motoristas (
-    id serial not null,
-    nome varchar(100),
-    senha varchar(50),
-    usuario_id bigint,
-    primary key (id)
-)
\ No newline at end of file
Index: LOGIDUTRA-master/src/main/resources/db/migration/V2__createtable.sql
===================================================================
diff --git a/LOGIDUTRA-master/src/main/resources/db/migration/V2__createtable.sql b/LOGIDUTRA-master/src/main/resources/db/migration/V2__createtable.sql
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/resources/db/migration/V2__createtable.sql	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,5 +0,0 @@
-create table pessoas (
-    id serial not null,
-    nome varchar(100)
-)
-
Index: LOGIDUTRA-master/src/main/java/org/example/models/Usuarios.java
===================================================================
diff --git a/LOGIDUTRA-master/src/main/java/org/example/models/Usuarios.java b/LOGIDUTRA-master/src/main/java/org/example/models/Usuarios.java
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/java/org/example/models/Usuarios.java	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,48 +0,0 @@
-package org.example.models;
-import jakarta.persistence.Entity;
-import jakarta.persistence.Id;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-
-@Entity(name = "usuario")
-
-public class Usuarios {
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)//isso aqui é tipo a query id serial la do sql que gera o id automaticamente
-    Long id;
-    private String usuario;
-    private String senha;
-
-    public Usuarios () {}; // contrutor sem parametros
-
-    public Usuarios(Long id, String usuario, String senha) {
-        this.id = id;
-        this.usuario = usuario;
-        this.senha = senha;
-    }
-
-    public String getUsuario() {
-        return usuario;
-    }
-
-    public void setUsuario(String usuario) {
-        this.usuario = usuario;
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public String getSenha() {
-        return senha;
-    }
-
-    public void setSenha(String senha) {
-        this.senha = senha;
-    }
-
-}
Index: LOGIDUTRA-master/pom.xml
===================================================================
diff --git a/LOGIDUTRA-master/pom.xml b/LOGIDUTRA-master/pom.xml
deleted file mode 100644
--- a/LOGIDUTRA-master/pom.xml	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,38 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <groupId>org.example</groupId>
-    <artifactId>untitled</artifactId>
-    <version>1.0-SNAPSHOT</version>
-
-    <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-
-    <dependencies>
-        <dependency>
-            <groupId>org.hibernate.orm</groupId>
-            <artifactId>hibernate-core</artifactId>
-            <version>7.3.0.Final</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>jakarta.persistence</groupId>
-            <artifactId>jakarta.persistence-api</artifactId>
-            <version>3.2.0</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.postgresql</groupId>
-            <artifactId>postgresql</artifactId>
-            <version>42.7.10</version>
-            <scope>compile</scope>
-        </dependency>
-    </dependencies>
-
-</project>
\ No newline at end of file
Index: LOGIDUTRA-master/src/main/java/org/example/models/Motoristas.java
===================================================================
diff --git a/LOGIDUTRA-master/src/main/java/org/example/models/Motoristas.java b/LOGIDUTRA-master/src/main/java/org/example/models/Motoristas.java
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/java/org/example/models/Motoristas.java	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,60 +0,0 @@
-    package org.example.models;
-
-    import jakarta.persistence.*;
-    import org.hibernate.sql.results.internal.StandardRowReader;
-
-    @Entity
-    @Table (name = "motorista")
-
-    public class Motoristas {
-
-        @OneToOne//relacionamento um pra um
-        @JoinColumn (name = "usuario_id")//cria uma coluna na tabela motoristas que guarda o id do usuario relacionado
-        private Usuarios usuarios;
-
-        public Usuarios getUsuarios() {
-            return usuarios;
-        }
-
-        public void setUsuarios(Usuarios usuarios) {
-            this.usuarios = usuarios;
-        }
-
-        @Id
-        @GeneratedValue (strategy = GenerationType.IDENTITY)
-        long id;
-        private String nome;
-        private String senha;
-
-        public Motoristas () {};
-
-        public Motoristas(long id, String nome, String senha) {
-            this.id = id;
-            this.nome = nome;
-            this.senha = senha;
-        }
-
-        public long getId() {
-            return id;
-        }
-
-        public void setId(long id) {
-            this.id = id;
-        }
-
-        public String getNome() {
-            return nome;
-        }
-
-        public void setNome(String nome) {
-            this.nome = nome;
-        }
-
-        public String getSenha() {
-            return senha;
-        }
-
-        public void setSenha(String senha) {
-            this.senha = senha;
-        }
-    }
Index: LOGIDUTRA-master/src/main/resources/hibernate.cfg.xml
===================================================================
diff --git a/LOGIDUTRA-master/src/main/resources/hibernate.cfg.xml b/LOGIDUTRA-master/src/main/resources/hibernate.cfg.xml
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/resources/hibernate.cfg.xml	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE hibernate-configuration PUBLIC
-        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
-        "https://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
-<hibernate-configuration>
-    <session-factory>
-        <property name="hibernate.connection.driver_class">org.postgresql.Driver</property>
-        <property name="hibernate.connection.url">jdbc:postgresql://localhost:5432/Roterizador</property>
-        <property name="hibernate.connection.username">postgres</property>
-        <property name="hibernate.connection.password">1234</property>
-
-        <property name="hibernate.show_sql">true</property>
-        <property name="hibernate.format_sql">true</property>
-        <property name="hibernate.hbm2ddl.auto">update</property>
-
-        <mapping class="org.example.models.Usuarios"/>
-    </session-factory>
-</hibernate-configuration>
\ No newline at end of file
Index: LOGIDUTRA-master/src/main/java/org/example/Repository/UsuarioRepository.java
===================================================================
diff --git a/LOGIDUTRA-master/src/main/java/org/example/Repository/UsuarioRepository.java b/LOGIDUTRA-master/src/main/java/org/example/Repository/UsuarioRepository.java
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/java/org/example/Repository/UsuarioRepository.java	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,43 +0,0 @@
-package org.example.Repository;
-
-import jakarta.persistence.EntityManager;
-import java.util.List;
-import org.example.models.Usuarios;
-
-public class UsuarioRepository {
-    private EntityManager em;//EntityManager serve pra interagir com banco de dados, o "em" é um atributo que eu defini do EntityManager
-
-    public UsuarioRepository(EntityManager em) {this.em = em;} //construtor da classe, o atributo "em" da classe vai receber o valor do "em" que veio no construtor
-
-    public Usuarios findbyId(Long id) { return (Usuarios) this.em.find(Usuarios.class, id);}//Busca de usuario pelo ID, o "find" é o metodo de busca
-
-    public void create (Usuarios usuarios) {
-        this.em.getTransaction().begin();//começa a operação
-        this.em.persist(usuarios);// faz a operação
-        this.em.getTransaction().commit();// vai salva caso de certo, se der errado cancela tudo, basicamente nao fica em um meio termo
-    }
-
-    public void update (Usuarios usuarios) {
-        em.getTransaction().begin();
-        em.persist(usuarios);
-        em.getTransaction().commit();
-    }
-
-    public void delete (Usuarios usuarios) {
-        em.getTransaction().begin();
-        em.remove(em.contains(usuarios) ? usuarios : em.merge(usuarios));/* o "em.contains" vai verifica se o objeto é gerenciado pelo EntityManager
-                                                                         usa o operador ternary pra verificar, se ja esta gerenciado usa usuarios e remove
-                                                                         se não esta sendo gerenciado faz o merge com o usuarios e ai remove*/
-        em.getTransaction().commit();
-    }
-
-    public List<Usuarios> findAll() { return em.createQuery("select u from Usuarios u", Usuarios.class).getResultList();}/* busca todos os usuarios do banco de dados
-                                                                                                                            e retorna uma lista*/
-    public List<Usuarios> findByName(String prefixo) {//esse prefixo é so o nome da variavel, passamos ela como parametro
-        return  em.createQuery("select u from Usuarios u where u.nome like :prefixo", Usuarios.class)// o like é um metodo de busca flexivel
-
-                .setParameter("prefixo", prefixo + "%")//o prefixo antes do "%" define que "começa com" ex: digitou "ma" vai exibir mateus, maria, madalena etc..
-
-                .getResultList();
-    }
-}
Index: LOGIDUTRA-master/src/main/java/org/example/Main.java
===================================================================
diff --git a/LOGIDUTRA-master/src/main/java/org/example/Main.java b/LOGIDUTRA-master/src/main/java/org/example/Main.java
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/java/org/example/Main.java	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,135 +0,0 @@
-package org.example;
-import org.example.models.Usuarios;
-import org.example.repositories.CustomizerFactory;
-import org.example.Repository.UsuarioRepository;
-import jakarta.persistence.EntityManager;
-import java.util.List;
-import java.util.Scanner;
-
-public class Main {
-    public static void main(String[] args) {
-        EntityManager em = CustomizerFactory.getEntityManager();// aqui cria o entitymanager que vai conversar com o banco
-        UsuarioRepository usuarioRepository = new UsuarioRepository(em);// o "em" é o EntityManager que será usado pelo repository para manipular o banco de dados
-        // chamando os metodos
-        novoUsuario(usuarioRepository);
-
-        exibirUsuarios(usuarioRepository);
-
-        alterarSenha(usuarioRepository);
-
-        removerUsuario(usuarioRepository);
-
-    }
-
-    static void novoUsuario(UsuarioRepository usuarioRepository) {//sem passa os parametros o metodo nao conseguiria acessar as variaveis de instancia
-
-        while (true) { //depois ajusta isso
-            Scanner sc = new Scanner(System.in);
-
-            System.out.println("digite S para sair do loop ou C para cadastrar");
-            String condição = sc.nextLine();
-
-            if (condição.equals("S") || condição.equals("s") ){
-                break;
-            }
-            if (condição.equals("C") || condição.equals("c")) {
-                Usuarios novo = new Usuarios();
-                System.out.println("\nCADASTRO DE USUARIO");
-
-                System.out.println("Usuario: ");
-                novo.setUsuario(sc.nextLine());
-
-                System.out.println("Senha: ");
-                novo.setSenha(sc.nextLine());
-
-                usuarioRepository.create(novo);// substitui o lista.add(novo) pelo create do banco de dados
-
-                System.out.println("USUARIO CADASTRADO");
-
-            }
-        }
-    }
-
-    static void exibirUsuarios(UsuarioRepository usuarioRepository) {
-        List<Usuarios> lista = usuarioRepository.findAll();//faz a busca de todos os usuarios de todos os usuarios do banco de dados e retorna uma lista
-        for (Usuarios u : lista) {
-            System.out.println("Usuario: " + u.getUsuario());
-        }
-    }
-
-    static void alterarSenha (UsuarioRepository usuarioRepository) {
-        Scanner sc = new Scanner(System.in);
-
-        System.out.println("Digite o Usuario que deseja atualizar a senha");
-        String user = sc.nextLine();
-
-        boolean encontrado = false;
-        String senhaDigitada;
-        Usuarios usuarioencontrado = null;
-
-        while (encontrado == false) {
-            usuarioencontrado = null;//caso o usuario nao seja encontrado, reseta o valor pra null;
-            List<Usuarios> lista = usuarioRepository.findAll();
-            for (Usuarios u : lista) {
-                if (u.getUsuario().equals(user)){
-                    usuarioencontrado = u;
-                }
-            }
-                if (usuarioencontrado != null) {
-                    if (usuarioencontrado.getUsuario().equals(user)) {// verificação desnecessaria, mas rodou assim entao deixei
-                        System.out.println("Senha antiga: ");
-                        senhaDigitada = sc.nextLine();
-
-                        if (senhaDigitada.equals(usuarioencontrado.getSenha())) {
-                            System.out.println("Nova Senha: ");
-                            String novaSenha = sc.nextLine();
-                            usuarioencontrado.setSenha(novaSenha);
-                            usuarioRepository.update(usuarioencontrado);// atualizando no banco de dados
-                            encontrado = true;
-
-                            System.out.println("Senha atualizada!!");
-                        }
-                    }
-                }else {
-                    System.out.println("Usuario nao encontrado, digite um usuario valido!!");
-                    user = sc.nextLine();
-                }
-            //quando as duas forem verdadeiras vai significar que  senha ta errada mas o usuario ta correto, evita de cair direto na senha incorreta
-            if (encontrado == false && usuarioencontrado != null) {
-                System.out.println("Senha incorreta!!\nDigite novamente");
-                senhaDigitada = sc.nextLine();
-            }
-        }
-    }
-
-    static void removerUsuario (UsuarioRepository usuarioRepository) {
-        Scanner sc = new Scanner(System.in);
-
-        System.out.println("Digite o usuario que sera removido: ");
-        String user = sc.nextLine();
-
-        Usuarios usuarioencontrado = null;
-        boolean encontrado = false;
-
-        while (encontrado == false) {
-            usuarioencontrado = null;
-            List<Usuarios> lista = usuarioRepository.findAll();
-
-            for (Usuarios u : lista) {
-                if (u.getUsuario().equals(user)) {
-                    usuarioencontrado = u;
-                }
-            }
-            if (usuarioencontrado!= null){
-                usuarioRepository.delete(usuarioencontrado);
-                encontrado = true;
-                System.out.println("Usuario deletado com sucesso!!");
-            }
-            {
-                System.out.println("Usuario não encontrado!!\n" +
-                        "Digite um usuario valido: ");
-                user = sc.nextLine();
-            }
-        }
-    }
-}
\ No newline at end of file
Index: LOGIDUTRA-master/src/main/java/org/example/config/FlyWayconfig.java
===================================================================
diff --git a/LOGIDUTRA-master/src/main/java/org/example/config/FlyWayconfig.java b/LOGIDUTRA-master/src/main/java/org/example/config/FlyWayconfig.java
deleted file mode 100644
--- a/LOGIDUTRA-master/src/main/java/org/example/config/FlyWayconfig.java	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ /dev/null	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
@@ -1,18 +0,0 @@
-package org.example.config;
-
-import org.flywaydb.core.Flyway;
-
-public class FlyWayconfig {
-    public static void migrate () {
-        Flyway flyway = Flyway.configure()
-                .dataSource(
-                        "jdbc:postgresql://localhost:5432/Roterizador",
-                        "postgres",
-                        "1234"
-                )
-                .baselineOnMigrate(true)
-                .load();
-
-        flyway.migrate();
-    }
-}
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"VcsDirectoryMappings\">\r\n    <mapping directory=\"$PROJECT_DIR$\" vcs=\"Git\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
--- a/.idea/vcs.xml	(revision d0b765c1d2d8dbbe1b449197b66ac7ae2a8e7437)
+++ b/.idea/vcs.xml	(date 1775687748692)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="" vcs="Git" />
   </component>
 </project>
\ No newline at end of file
diff --git a/LOGIDUTRA-master/src/main/resources/db/migration/V1__create_database.sql b/LOGIDUTRA-master/src/main/resources/db/migration/V1__create_database.sql
deleted file mode 100644
